app/views/organized_mailer/organization_invitation.text.haml

```diff
@@ -1,4 +1,4 @@
-- if @new_user = true
+- if @new_user == true
```

Comment:
Yikes, thanks for catching this 🙏

Reply:
I can't take credit. It's all from the spec output!

Now that we have less garbage in there, I'm going to check in there periodically, since there are useful things 🙂